src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Markup/MarkupExtensionParser.cs

```diff
@@ -1625,7 +1626,9 @@ internal static void RemoveEscapes(ref string value)
                     if (builder == null)
                     {
                         builder = new StringBuilder(value.Length);
+#pragma warning disable CA1846
                         builder.Append(value.Substring(0,i));
```

Comment:
```suggestion
                        builder.Append(value, 0, i));
```
This overload is available on all platforms and will do the performant thing on .NET Core.

Reply:
Thank you for the suggestion @h3xds1nz 😄. Added.